Controllers/OrderController.js

```diff
@@ -41,4 +46,119 @@ let createNewOrder = (req, res) => {
     })
 }
 
-module.exports.createNewOrder = createNewOrder
\ No newline at end of file
+const addToOrder = (req, res) => {
+    DbService.connectToDb(async (db) => {
+        const itemsToOrder = {
+            orderId: req.body.orderId,
+            orderItems: req.body.orderItems
+        }
+
+        const compile = ajv.compile(addToOrderValidate)
+        const valid = compile(itemsToOrder)
+
+        if (valid) {
+            try {
+                await OrderService.addItemsToOrder(db, itemsToOrder)
+                let response = JSONResponseService.generateSuccessResponse()
+                response.message = "Dish successfully added to order"
+                return res.json(response)
+            } catch (e) {
+                let response = JSONResponseService.generateFailureResponse()
+                response.message = "Dish not found so cannot add to order"
+                return res.json(response)
+            }
+        }
+        let response = JSONResponseService.generateFailureResponse()
+        response.message = "Invalid data types provided"
+        return res.json(response)
+    })
+}
```

Comment:
How did this end up in your branch as an insertion? This is the function from story4

Reply:
I think it was moved around in the merge conflicts